compiler/main/commonFlags.h

```diff
@@ -41,6 +41,10 @@ void driverSetDevelSettings(const ArgumentDescription* desc, const char* arg_unu
 #define DRIVER_ARG_COPYRIGHT \
   {"copyright", ' ', NULL, "Show copyright", "F", &fPrintCopyright, NULL, NULL}
 
+#define DRIVER_ARG_BREAKFLAGS_COMMON \
```

Comment:
You might consider lining up the two backslashes in this macro like they are in the next macro.  The downside is that it would push both lines past 80 characters instead of just one.